framework/src/base/FEProblem.C

```diff
@@ -515,12 +515,24 @@ void FEProblem::initialSetup()
   // HUGE NOTE: MultiApp initialSetup() MUST... I repeat MUST be _after_ restartable data has been restored
 
   // Call initialSetup on the MultiApps
+  bool has_multiapp = false;
+  if (_multi_apps(EXEC_LINEAR)[0].all().size() ||
```

Comment:
Why not just set this Boolean when you we add a Multiapp to the system? I could have sworn that we already had a Boolean for this...